src/cpu.cpp

```diff
@@ -847,4 +851,49 @@ void set_kmp_blocktime(int time_ms)
 #endif
 }
 
+static int g_flush_denormals = 0;
```

Comment:
The global variable will cause data-race in multi-threaded inference jobs. We always modify this state each time Extractor::extract() called, and multiple threads may call it concurrently.

Use ThreadLocalStorage in platform.h and set/get functions.

```
static ncnn::ThreadLocalStorage tls_flush_denormals;
```

some code samples
https://github.com/Tencent/ncnn/blob/9fd4d371ae03a70a73162ecde624c7feb8c0a82b/src/simpleomp.cpp#L294
https://github.com/Tencent/ncnn/blob/9fd4d371ae03a70a73162ecde624c7feb8c0a82b/src/simpleomp.cpp#L304